subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -825,6 +825,7 @@ int bt_mesh_cfg_app_key_add(u16_t net_idx, u16_t addr, u16_t key_net_idx,
 
 	err = cli_prepare(&param, OP_APP_KEY_STATUS);
 	if (err) {
+		printk("err %d\n", err);
```

Comment:
Should be `BT_ERR()` or `BT_DBG()`

Reply:
Oh that's just something left from debugging, will remove it.